compendium/modules/w05-classes-exercise.tex

```diff
@@ -244,7 +244,7 @@
   def distanceTo(that: (Int, Int)): Double = ???
 
   /** A new Point that is moved (dx, dy) */
-  def move(dxdy: (Int, Int)): Point = ???
+  def move(dx: Int, dy: Int): Point = ???
```

Comment:
Since an `(Int, Int)` is used elsewhere in the exercise so that's probably the way to go. If the solution uses `dx: Int, dy: Int` then that should be changed to `dxdy: (Int, Int)`.